### PR TITLE
Added OpenSpeedTest to Miscellaneous

### LINKED ISSUE #3119: Add OpenSpeedTest (HTML5 SpeedTest)
https://github.com/openspeedtest/Speed-Test
https://hub.docker.com/r/openspeedtest/latest
HTML5 Network Network Speed Test
No client-side software or plugin is required. You can run a network speed test from any device with a web browser that is IE10 or new.

### PR DESCRIPTION
**Pull request by CrimsonTome**
Closes #3119
Source Code: https://github.com/openspeedtest/Speed-Test
Installation: https://github.com/openspeedtest/Speed-Test#docker-install-instructions

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ ] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1264,6 +1264,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Ombi](https://ombi.io/) - A content request system for Plex/Emby, connects to SickRage, CouchPotato, Sonarr, with a growing feature set. ([Demo](https://app.ombi.io/), [Source Code](https://github.com/Ombi-app/Ombi)) `GPL-2.0` `C#`
 - [OpenZiti](https://openziti.github.io/) - Fully-featured, self-hostable, zero trust overlay network. Includes a full mesh overlay network, includes 2FA support out of the box, includes clients for all major desktop/mobile OS'es. Build zero trust network principles into your application via SDKs. ([Source Code](https://github.com/openziti/ziti)) `Apache-2.0` `Go`
 - [Orchest](https://www.orchest.io/) - A new kind of IDE for Data Science. ([Demo](https://cloud.orchest.io), [Source Code](https://github.com/orchest/orchest)) `AGPL-3.0` `Docker`
+- [OpenSpeedTest](https://github.com/openspeedtest/Speed-Test) - Pure HTML5 network performance estimation tool. `CC BY-ND 3.0` `Docker`
```
Comment (KieranRobson) [inline README.md:1267]: Congratulations on your cotribution!

- One small change is that you could change the main link to `https://openspeedtest.com/` and add a seperate [Source Code](https://github.com/openspeedtest/Speed-Test) link

Comment (CrimsonTome) [reply README.md:1267]: Thanks, I will do that now :smile:

Comment (KieranRobson) [reply README.md:1267]: Another change is to change the license tag to `CC-BY-ND-3.0` inline with other entries within the list

**Comment by CrimsonTome** [comment]
Changes should have been done now, noticed the license is not in the list however, oops

**Comment by KieranRobson** [comment]
You are missing a pair of brackets surround Source code so that it becomes (Source Code) rather than Source Code

File: README.md [hunk 2]
```diff
@@ -1264,6 +1264,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Ombi](https://ombi.io/) - A content request system for Plex/Emby, connects to SickRage, CouchPotato, Sonarr, with a growing feature set. ([Demo](https://app.ombi.io/), [Source Code](https://github.com/Ombi-app/Ombi)) `GPL-2.0` `C#`
 - [OpenZiti](https://openziti.github.io/) - Fully-featured, self-hostable, zero trust overlay network. Includes a full mesh overlay network, includes 2FA support out of the box, includes clients for all major desktop/mobile OS'es. Build zero trust network principles into your application via SDKs. ([Source Code](https://github.com/openziti/ziti)) `Apache-2.0` `Go`
 - [Orchest](https://www.orchest.io/) - A new kind of IDE for Data Science. ([Demo](https://cloud.orchest.io), [Source Code](https://github.com/orchest/orchest)) `AGPL-3.0` `Docker`
+- [OpenSpeedTest](https://github.com/openspeedtest/Speed-Test) - Pure HTML5 network performance estimation tool. ([Source Code](https://github.com/openspeedtest/Speed-Test)) `CC-BY-ND 3.0` `Docker`
```
Comment (nodiscc) [inline README.md:1267]: >  Do not add a duplicate Source code link if it is the same as the main link.

Please remove the source code link

> This is a list of [Free](https://en.wikipedia.org/wiki/Free_software) Software
>> The freedom to improve the program, and release your improvements (and modified versions in general)

> https://creativecommons.org/licenses/by-nd/3.0/
>> NoDerivatives — If you [remix, transform, or build upon](https://creativecommons.org/licenses/by-nd/3.0/#) the material, you may not distribute the modified material.

`CC-BY-ND-3.0` is not a Free/Libre license. OpenSpeedTest must be moved to `non-free.md`, unless the license is changed. Have you considered switching to a license that allows modifications/derived works?

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by CrimsonTome** [comment]
I beleve I have fixed the outstanding issues, please let me know if there is anything else I need to do. :smile:

**Review by nodiscc — APPROVED** [review]

**Comment by joho1968** [comment]
Out of curiosity, where is it "non-free"?

**Comment by KieranRobson** [comment]
> Out of curiosity, where is it "non-free"?

While the actual software may not cost you actual money to use, it is not "[Free Software](https://en.m.wikipedia.org/wiki/Free_software)"

**Pull request merged**